test/k8s/simple_test.go

```diff
@@ -13,22 +20,40 @@ import (
 
 func Simple() {
 	BeforeAll(func() {
+		opts := []KumaDeploymentOption{
+			WithInstallationMode(HelmInstallationMode),
+			WithHelmChartVersion(meshVersion),
+			WithHelmReleaseName(fmt.Sprintf("kuma-%s", strings.ToLower(random.UniqueId()))),
+			WithHelmChartPath(Config.HelmChartName), // we pass chart name to use production chart
+			WithoutHelmOpt("global.image.tag"),      // required to use production chart
+		}
+
+		if license := os.Getenv("KMESH_LICENSE_INLINE"); license != "" {
```

Comment:
Should we add `Expect(license).ToNot(BeEmpty())`? Otherwise, it won't work, right?

Reply:
It can work with up to 5 services. Additionally, we may be able to run this with Kuma in the future so I don't want the license to be required.

Reply:
But this `should deploy graph` test won't pass without the license right?

Reply:
Oh I see, it has 5 services